src/components/analyses/listing/Listing.js

```diff
@@ -115,9 +125,10 @@ function Listing(props) {
             .join(",");
 
         setAnalysesKey([
-            "getAnalyses",
+            ANALYSES_LISTING_QUERY_KEY,
```

Comment:
I only did a quick look-over of the changes in v2, but it seems like with the introduction to the `enabled` flag, we don't need to keep the keys in state?  I thought we were keeping the keys in state initially because the previous method was to have a falsy query key to prevent a query from running.  Now it seems like we can remove that and rely on the `enabled` flag.

Reply:
I think we will still need to keep the key in state at least for analyses and app listings. For eg: in analyses listing, the key contains additional parameters like `pagesize`, `offset`, `filters` etc. Although, all these parameters are in state, the `filter` needs little bit of processing before we fire off the query. That happens between line `86 - 125` in `/analyses/listing/Listing.js`. For data listing, we can get away with not having the `key` in state but it does include `uploadsCompleted` which forces a `refresh`. Not sure if there is another approach to managing these keys.

Reply:
This is ultimately minor, so feel free to disregard. If I understand correctly, you can control when a query runs by either managing when the query key gets set or setting the enabled flag to true/false. And instead of picking one of those methods, it looks like we're kind of doing both is what I'm saying.

I haven't looked too deeply at the code, so it may be the case that removing the keys from state and moving the components of the keys into state (like the filters you were talking about since `page`, `orderBy`, etc. are already in state) would end up being the same number of things in state. In that case, it doesn't really make a difference and definitely disregard my comment.